.travis.yml

```diff
@@ -2,6 +2,8 @@ os: linux
 dist: xenial
 language: php
 
+if: type != push OR branch = master OR branch =~ /^v?\d+\.\d+(\.\d+)?(-\S*)?$/
```

Comment:
Not sure if this optimisation is worth it, considering that the repo doesn't get frequent/daily contributions

Reply:
I got annoyed by Dependabot running CI twice on every opened PR/rebase too much maybe.

And well, it's also good for the planet™

Reply:
Makes sense, but you shall be pinged if this doesn't work as expected :-P

Reply:
And I'll answer the call :sunglasses:

Reply:
Thanks! :heart: